File: MethodOR.java
class parent{
    void num(){
        System.out.println("10 is the number");
    }
}
class son extends parent{
    void num(){
        System.out.println("20 is the number");
    }
}
class override{
    public static void main(String[] args){
        parent a = new parent();
        son b = new son();
        a.num();
        b.num();

    }
}

File: MethodOR.cpp
#include<iostream>
using namespace std;
class parent{
    public:
    void num(){
        cout<<"10 is the number"<<endl;
    }
};
class son :public parent{
    public:
    void num(){
        cout<<"20 is the number"<<endl;
    }
};
int main(){
    parent a;
    son b;
    a.num();
    b.num();

}
